lib/puppet/provider/package/aix.rb

```diff
@@ -96,6 +105,10 @@ def install(useversion = true)
     if output =~ /^#{Regexp.escape(@resource[:name])}\s+.*\s+Already superseded by.*$/
       self.fail _("aix package provider is unable to downgrade packages")
     end
+
+    if [:broken, :inconsistent].include?(query[:status])
+      self.fail _("Installed package is in a broken or inconsistent state")
```

Comment:
Would it be more helpful to add the actual status instead of `broken or inconsistent`?
```ruby
self.fail _("Installed package is in a #{query[:status]} state")
```